File: LinearSearch.php
<?php
$arr=array(10,20,30,40,50);
$key=40;
for($i=0;$i<=sizeof($arr);$i++)
{
	if($key==$arr[$i])
		break;
}
if($i<sizeof($arr))
{$i=$i+1;
echo("Element found at position:".$i);
}else
echo "Element not found";
?> 

File: LinearSearch.java
import java.lang.*;
class LinearSearch{
int i,n=5,key=40;
int a[]={10,20,30,40,50};
int search()
{
for(i=0;i<n;i++)
{
if(key==a[i])
return i+1;
}
return -1;
}

public static void main(String[] args){
LinearSearch obj1=new LinearSearch();
int k=obj1.search();
if(k==-1)
System.out.println("Element not found");
else
System.out.println("Element found at position:"+k);
}
}